src/admin/menu/menu.const.ts

```diff
@@ -22,3 +24,14 @@ export const MENU_ICON_OPTIONS: OptionsType<ReactSelectOption<IconName>> = Objec
 	{ label: 'Aktetas', value: 'briefcase' },
 	{ label: 'Zoek', value: 'search' },
 ]);
+
+export const INITIAL_MENU_FORM: MenuEditForm = {
+	icon: '',
+	label: '',
+	link: '',
+};
+
+export const PAGE_TYPES_LANG = {
```

Comment:
Als je hier een enum van maakt dan heb je die `MenuEditPageType ` niet nodig
Deze dan wel in de .types file zetten

Reply:
nvm, ik zie dat je die gebruikt voor de pageTitle op te bouwen